roles/rpm_uninstall/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# vim: set ft=ansible:
+#
+# Removes package using rpm
+#
+- name: Fail if g_rpm_name
+  fail:
+    msg: "g_rpm_name is undefined"
+  when: g_rpm_name is undefined
+
+- name: Install {{ g_rpm_name }}
+  command:  rpm -e {{ g_rpm_name }}
```

Comment:
I feel like this role should also be using `rpm_name` as the variable.  I know it is not currently used with a different value than `g_rpm_name` but we should provide that flexibility.